chapters/improvements/sections/world_improvements.tex

```diff
@@ -0,0 +1,17 @@
+\section{World improvements}
+The world could be improved in multiple ways.
+One such way would be to add shadows to the game.
+Right now nothing casts any shadows but it would be interesting to see what shadows look like in non-Euclidean geometries.
+
+Another improvement to the world would be to add clouds.
+The sky is very empty during the day with only the suns visible.
+Its better during the night but nevertheless clouds would make it look even better.
+
+Perhaps the most important improvement to the world would be to add sound.
+Right now the game is completely silent which makes it feel empty.
+Adding a soundtrack would make tha game more pleasant to play and adding sound effects would make the game feel more alive.
```

Comment:
Maybe write something about the fact that we already have a system for collision callbacks and it could be used here easily (when a projectile hits a player or something like that)